Python/Python.sublime-syntax

```diff
@@ -1016,8 +1016,17 @@ contexts:
 
   assignments:
     - include: illegal-assignment-expression
-    - match: ':'
-      scope: punctuation.separator.annotation.variable.python
+    - match: '(?=:)'
+      set:
```

Comment:
1. This would need to be a `push` as `assignments` is included in `statements` which is part of `main`.

2. Why adding the lookahead?

Reply:
Both because it is copied from `function-parameters: `.  I push a change so we can compare.  Don't know if I get this to work actually.

Reply:
Ok, I pushed a commit and it checks green.